sonata-project/media-orm-pack/1.0/post-install.txt

```diff
@@ -0,0 +1,4 @@
+<bg=blue;fg=white> Integration SonataMediaBundle with SonataClassificationBundle </>
+
+  * If you are using <info>sonata-project/classification-bundle</> fix annotations for
```

Comment:
I think you need a `,` after the bundle name. What does our spellcheck say, summoning @greg0ire 😄